inspirehep/modules/multieditor/utilities.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2017 CERN.
+#
+# INSPIRE is free software; you can redistribute it
+# and/or modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be
+# useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE; if not, write to the
+# Free Software Foundation, Inc., 59 Temple Place, Suite 330, Boston,
+# MA 02111-1307, USA.
+#
+# In applying this license, CERN does not
+# waive the privileges and immunities granted to it by virtue of its status
+# as an Intergovernmental Organization or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, print_function, division
+
+
+from invenio_records.api import Record
+from inspirehep.modules.migrator.tasks import chunker
+
+
+def filter_records(uuids, filters):
+    '''
```

Comment:
```s/'''/"""```

Reply:
Multillne docstrings it's good to follow [this](https://www.python.org/dev/peps/pep-0257/#id17) ☺️

Reply:
in general for docstrings [example](http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html)
```python
"""Filter records with the given filters.

Args:
    uuids (list): a list of record uuids.
    filters (list): a list of filters.

Note:
     Filters should have the follow format:
     
                {
                     format: {
                            keypath: '',
                            value: ''     
                    }
              }

Returns:
     int: the number of total matches.
"""
```

Reply:
Got it :)